style.css

```diff
@@ -0,0 +1,78 @@
+/* nejsme vcerejsi, pouzivame moderni box model */
+
+html {
+    box-sizing: border-box;
+}
+
+*,
+ :after,
+ :before {
+    box-sizing: inherit;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 14px;
+}
+
+
+/* ---------------------------------------------------- */
+
+.zeny {
+    display: flex;
+    background-color: rgb(211, 211, 211);
+    border-radius: 30px;
+    max-width: 1300px;
+    margin: 2rem;
+}
+
+.hanka {
+    display: block;
+    width: 22%;
+    padding: 2rem;
+    margin: 2rem;

```

Comment:
Tady pozor na to, jak margin nastavuješ. Když se podíváš na sousedící položky, tak uvnitř odsazení nesedí - je dvojnásobné, protože se sčítá pravý margin levé položky s levým marginem pravé položky. Takže tady je potřeba nastavit odsazení tak, aby vycházela všude správně. Například nastavit margin na 1rem a padding na kontejneru .zeny zprava a zleva taky na 1rem, ať nám to všude sedí 🙂